packages/Pagination/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@tds/community-pagination",
+  "version": "1.0.0-preview",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telus/tds-community.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telus/tds-community/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": ">=15",
```

Comment:
Just a thought, what do you think about releasing this with a react `peerDependency` of `^16.8.2` this will put the component in line with starter kit as well as our styled-component rewrite?

```
"peerDependencies": {
  "react": "^16.8.2",
  "react-dom": "^16.8.2"
}
```